src/devices_models/devices/hydro_generation.jl

```diff
@@ -185,20 +185,23 @@ function _get_time_series(
         bus_number = PSY.get_number(PSY.get_bus(device))
         name = PSY.get_name(device)
         tech = PSY.get_tech(device)
-        # Hydro gens dont't have a power factor field, so the pf calc is commented
+        # Hydro gens dont't have a power factor field, so the pf is calculated
         # pf = sin(acos(PSY.get_powerfactor(PSY.get_tech(device))))
-        active_power =
-            use_forecast_data ? PSY.get_rating(tech) : PSY.get_activepower(device)
-        reactive_power =
-            use_forecast_data ? PSY.get_rating(tech) : PSY.get_reactivepower(device)
+        pf = PSY.get_reactivepower(device) / PSY.get_rating(tech) # TODO: remove if pf gets added to hydro struct
```

Comment:
we don't have a constant power factor model for Hydro. The reactive power modeling for hydro must follow something similar to thermal. 

The key aspect to consider in Hydro modeling is that we can generally (except when hydro storage and reservoir coordination is present) think of it as renewable in terms of active power and as thermal in terms of reactive power.